samples/bluetooth/central_hids/src/main.c

```diff
@@ -176,10 +188,71 @@ static void scan_init(void)
 	}
 }
 
+static u8_t hids_c_notify_cb(struct bt_gatt_hids_c *hids_c,
+			     struct bt_gatt_hids_c_rep_info *rep,
+			     u8_t err,
+			     const u8_t *data)
+{
+	u8_t size = rep->size;
+	u8_t i;
+
+	if (!data) {
+		return BT_GATT_ITER_STOP;
+	}
+	printk("Notification, id: %u, size: %u, data:",
+	       (unsigned)rep->ref.id,
+	       (unsigned)size);
+	for (i=0; i<size; ++i) {
+		printk(" 0x%x", (unsigned)data[i]);
```

Comment:
cast is not needed as default argument promotion is in place

Reply:
Fixed